packages/blueprints/blueprint-builder/src/build-blueprint-package.ts

```diff
@@ -34,6 +34,7 @@ export function buildBlueprintPackage(
   });
 
   const newBlueprintOptions: ProjenBlueprintOptions = {
+    homepage: options.bpOptions.homepage,
```

Comment:
Is this `homepage` value actually used by the projen-blueprint? I don't see `homepage` defined in `ProjenBlueprintOptions`.